java/arcs/android/type/ParcelableType.kt

```diff
@@ -113,6 +114,31 @@ sealed class ParcelableType(open val actual: Type) : Parcelable {
         }
     }
 
+    /** [Parcelable] variant of [arcs.core.data.TupleType]. */
+    class TupleType(
+        override val actual: arcs.core.data.TupleType
+    ) : ParcelableType(actual) {
+        override fun writeToParcel(parcel: Parcel, flags: Int) {
+            super.writeToParcel(parcel, flags)
+            parcel.writeInt(actual.elementTypes.size)
+            for (element in actual.elementTypes) {
+                parcel.writeType(element, flags)
+            }
+        }
+
+        companion object CREATOR : Parcelable.Creator<TupleType> {
+            override fun createFromParcel(parcel: Parcel): TupleType {
+                val elements = mutableListOf<Type>()
+                for (i in 0 until parcel.readInt()) {
```

Comment:
Optional: This could be slightly simpler as:

```kotlin
repeat(parcel.readInt()) {
    elements.add(requireNotNull(parcel.readType()))
}
```

Reply:
Thanks! :)